doc/developer-notes.md

```diff
@@ -1089,11 +1089,11 @@ A few guidelines for introducing and reviewing new RPC interfaces:
     new RPC is replacing a deprecated RPC, to avoid both RPCs confusingly
     showing up in the command list.
 
-- Use *invalid* bech32 addresses (e.g. the constant `EXAMPLE_ADDRESS`) for
+- Use *invalid* addresses (e.g. in the constant array `EXAMPLE_ADDRESS`) for
```

Comment:
The "bech32" part is actually important here, as we want to get rid of legacy addresses in the long term. Without that the reader of the guideline may think that any address format is fine.